package.json

```diff
@@ -80,6 +79,6 @@
     "webpack": "4.32.2"
   },
   "dependencies": {
-    "core-js": "^2.6.8"
+    "core-js": "^3.31.1"
```

Comment:
we don't need polyfills anymore

Reply:
Thank you for the feedback, I have removed it.